tests/console/yast2_dnsserver_service_abort_keep.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Ensure that all combinations of running/stopped and active/inactive can be set for yast2 dns-server
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base "y2_module_consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use Utils::Architectures;
+
+sub run {
+    my $dnsserver = $testapi::distri->get_dns_server();
+
+    select_console 'root-console';
+    YaST::Module::open(module => 'dns-server', ui => 'ncurses');
+    $dnsserver->process_reading_configuration();
+    $dnsserver->select_start_after_writing_configuration();
+    save_screenshot;
+    $dnsserver->select_start_on_boot_after_reboot();
+    save_screenshot;
+    $dnsserver->press_abort();
```

Comment:
Probably we want to reflect with clarity the difference between apply and ok, so we can be explicit with method `->ok()` and `->apply()` `->cancel()` and `->help()`. wdyt?

Reply:
So you mean to update the method names?
Ex:
accept_apply() -> apply()
press_cancel() -> cancel()
accept_ok() -> ok()

Reply:
I think the current name is ok for me. :) Updated for the comments except  part of names(test suites, methods...), we can talk about them later to find better solution. 
Please continue review. Thanks.

Reply:
it is a matter of taste, with `apply` is enough to understand, instead of `accept_apply`, up to you :)
Next libyui-rest-api test shouldn't have Controllers, so we would save time inventing names :)

Reply:
Thanks, I will update the press_abort to accept_cancel to make it uniform with others.